packages/teams-js/src/public/interfaces.ts

```diff
@@ -153,20 +154,29 @@ export interface TeamInformation {
  * Represents OS locale info used for formatting date and time data
  */
 export interface LocaleInfo {
+  /** Represents the platform on which the app is running. */
   platform: 'windows' | 'macos';
+  /** Represents the regional format used by the user's locale. */
```

Comment:
>Represents the regional format used by the user's locale.  [](http://example.com/codeflow?start=6&length=58)

What format is this format in?

Reply:
Example regional format is `en-us`. I'll add an example to make it clearer.